cpp/src/bindings.cpp

```diff
@@ -634,9 +634,11 @@ PYBIND11_MODULE(_pypowsybl, m) {
             py::arg("new_line_id"), py::arg("new_line_r"), py::arg("new_line_x"), py::arg("new_line_b1"), py::arg("new_line_b2"), py::arg("new_line_g1"), py::arg("new_line_g2"),
             py::arg("line_id"), py::arg("line1_id"), py::arg("line1_name"), py::arg("line2_id"), py::arg("line2_name"), py::arg("position_percent"),
             py::arg("create_fictitious_substation"), py::arg("fictitious_voltage_level_id"), py::arg("fictitious_voltage_level_name"), py::arg("fictitious_substation_id"), py::arg("fictitious_substation_name"));
+    m.def("revert_create_line_on_line", &pypowsybl::revertCreateLineOnLine, "reverses the action done in the create_line_on_line", py::arg("network"), py::arg("lineToBeMerged1Id"), py::arg("lineToBeMerged2Id"), py::arg("lineToBeDeletedId"), py::arg("mergedLineId"), py::arg("mergedLineName"));
```

Comment:
I think it should be snake_case for the arguments ("line_to_be_merged1_id", ... )